app/models/form.rb

```diff
@@ -32,12 +32,22 @@ def has_live_version
     live? || live_with_draft?
   end
 
+  def has_been_archived?
```

Comment:
does this method get included in the json to forms-admin and does it get added to the snapshot. Just wondering why this ends with a question mark but `has_draft_version` and `has_live_version` doesn't